packages/contracts-bedrock/foundry.toml

```
@@ -55,6 +55,7 @@ ignore = ['src/vendor/WETH9.sol']
 
 [profile.ci.fuzz]
 runs = 512
+deny_warnings = true
```

The key here is nested under the `fuzz` key but it should be
```suggestion
[profile.ci]
deny_warnings = true

[profile.ci.fuzz]
runs = 512
```

Or use the more concise inline table syntax which is my preference
```suggestion
[profile.ci]
deny_warnings = true
fuzz = { runs = 12 }
```

---

good catch. Fixed per your pref.